app/services/template_kind_name_jar.rb

```diff
@@ -0,0 +1,21 @@
+class TemplateKindNameJar
+  def self.jar
+    @jar ||= { "PXELinux" => N_("PXE Linux"),
+               "PXEGrub" => N_("PXE Grub"),
+               "iPXE" => N_("iPXE"),
+               "provision" => N_("Provision"),
+               "finish" => N_("Finish"),
+               "script" => N_("Script"),
+               "user_data" => N_("User Data"),
+               "ZTP" => N_("ZTP"),
+               "POAP" => N_("POAP")
+             }
+  end
+
+  def self.add(hash)
+    hash.each do |key, value|
```

Comment:
you can use `Hash.merge` with a block to achieve the same result without looping over the keys. see http://ruby-doc.org/core-2.2.0/Hash.html#method-i-merge fore more info.